gitstrap.go

```diff
@@ -18,13 +17,22 @@ import (
 	"os/exec"
 	"strings"
 	"text/template"
+	"runtime"
+	"path/filepath"
 )
 
 const (
 	// V1 - first version of config
 	V1 = "v1"
 )
 
+var (
+    _, b, _, _ = runtime.Caller(0)
```

Comment:
@mohamednizar it won't work, since `caller` doesn't return current directory:
> Caller reports file and line number information about function invocations on the calling goroutine's stack. The argument skip is the number of stack frames to ascend, with 0 identifying the caller of Caller. (For historical reasons the meaning of skip differs between Caller and Callers.) The return values report the program counter, file name, and line number within the file of the corresponding call. The boolean ok is false if it was not possible to recover the information. 

E.g. if I compile go programm:
```go
package main

import (
        "fmt"
        "runtime"
)

func main() {
        _, b, _, _ := runtime.Caller(0)
        fmt.Println(b)
}
```
at `/tmp/1` location, and then start it from different directory it'll print `/tmp/1/` file location:
```bash
$ cd ~
$ /tmp/1/prog
/tmp/1/prog.go
$ pwd
/home/g4s8
```

Reply:
FIx the $PWD read issue associated with gopwd package. Pointed the folk https://github.com/mohamednizar/gopwd as well as created a pull request for https://github.com/g4s8/gopwd.

See the out put of test cases:

```
nizar@nizar-GL63-8SD:~/gopwd$ go run main.go 
gopwd <nil>

nizar@nizar-GL63-8SD:~$ go run gopwd/main.go 
nizar <nil>
```
Hope this would fix the issue #2